CODEOWNERS

```diff
@@ -102,6 +102,7 @@ lessmsi @mwrock
 visual-cpp-build-tools-2015 @mwrock
 visual-cpp-redist-2013 @mwrock
 wix @mwrock
+postgresql-client @joshbrand @irvingpop
```

Comment:
Could you move this to the Community Plans section?

Reply:
no problem @smacfarlane

Reply:
done